python_modules/dagster-graphql/dagster_graphql/implementation/fetch_sensors.py

```diff
@@ -166,7 +166,7 @@ def get_sensors_for_pipeline(
     results = []
     for external_sensor in external_sensors:
         if pipeline_selector.job_name not in [
-            target.job_name for target in external_sensor.get_external_targets()
+            target.job_name for target in external_sensor.get_targets()
```

Comment:
Since we're establishing new conventions, should this be `get_target_snaps` or `get_targets`?

Reply:
The convention I've been following in this stack is that accessors on a `Snap` class that return other snaps are _not_ qualified with "snap". Snaps are implicitly understood to contain other snaps.

Open to debate on this but IMO it makes for more readable code in a world where you can use editor hover to get type information for an accessor.